ui/pages/bridge/index.tsx

```diff
@@ -89,7 +89,7 @@ const CrossChainSwap = () => {
   return (
     <Page className="bridge__container">
       <Header
-        textProps={{ variant: TextVariant.headingSm }}
+        textProps={{ variant: TextVariant.headingSm, as: 'h1' }}
```

Comment:
Adding the as prop here changes the default `div` to `h1` and improves the semantic html and accessibility of the bridge page see screenshot in snapshot